### DISCUSSION TITLE
Prompt for virus and plasmid generation

### DISCUSSION THREAD

**Discussion by apcamargo** [evo-design/evo#74] [Q&A]
The example notebook show the utilization of a Greengenes-style lineage as a prompt for generation. However, it does not address the process of prompting for non-chromosomal sequences. For viruses, was the ICTV taxonomy from IMG/VR used during training? Additionally, concerning plasmids, if their host lineage was utilized, how can I direct Evo to generate a plasmid?

**Comment by brianhie** [comment]
>For viruses, was the ICTV taxonomy from IMG/VR used during training?

This is correct!

>Additionally, concerning plasmids, if their host lineage was utilized, how can I direct Evo to generate a plasmid?

During the 131k context extension phase, there was no species level prompt prepended to the plasmids.

Great questions!

**Comment by jproney** [comment]
Thanks for the great models! I was wondering if you could provide some examples of prompt strings with virus lineages -- in particular, what character is prepended to indicate the "Realm" and "Kingdom" classification levels used by ICTV? Also, in the paper and in various issues on GitHub the lineage prompts have two pipe symbols at the end (`||`), but in the `hello_evo.ipynb` notebook there is only one. Which is the correct format? Thanks again!
